### PR TITLE
chore: add isort to ci

### PR DESCRIPTION
**Pull request by feng-tao**
Signed-off-by: feng-tao <fengtao04@gmail.com>

### Summary of Changes

add isort to CI to sort import order based on pep8

### Tests

_What tests did you add or modify and why? If no tests were added or modified, explain why. Remove this line_

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes.
- [ ] PR adds unit tests, updates existing unit tests, **OR** documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
- [ ] PR passes `make test`

### CODE REVIEW

File: metadata_service/__init__.py [hunk 1]
```diff
@@ -7,26 +7,29 @@
...
-from flask import Flask, Blueprint
+from flask import Blueprint, Flask
+from flask_cors import CORS
 from flask_restful import Api
 
+from metadata_service.api.badge import BadgeAPI
 from metadata_service.api.column import ColumnDescriptionAPI
-from metadata_service.api.dashboard import (DashboardDetailAPI, DashboardDescriptionAPI,
-                                            DashboardTagAPI, DashboardBadgeAPI)
+from metadata_service.api.dashboard import (DashboardBadgeAPI,
```
Comment (mgorsk1) [inline metadata_service/__init__.py:19]: Maybe we could mąke things consistent regarding wrapping modules in tuple (now there is mix of both w/ and w/o) ? I am in favor of tuple and multiline (one module per line) imports as it reduces chances for clashes during merges and makes things może readable in overall.

**Review by mgorsk1 — COMMENTED** [review]
One suggestion only

File: metadata_service/__init__.py (hunk 1 above)
Comment (feng-tao) [reply metadata_service/__init__.py:19]: @mgorsk1 I will add the isort to all repos.

Comment (feng-tao) [reply metadata_service/__init__.py:19]: oh, I need to take a look at and see how to change isort for one module at a time though. I think it only handles import order.

**Review by allisonsuarez — APPROVED** [review]

**Pull request merged**